.github/workflows/circleci-artifacts-redirector.yml

```diff
@@ -0,0 +1,12 @@
+name: CircleCI Artifacts Redirector
```

Comment:
How this action works:

https://user-images.githubusercontent.com/17039389/179951667-7a21caf2-46f8-457a-a588-409cf9318dc0.mov